### PR TITLE
Feat/general ticket update

### PR DESCRIPTION
**Pull request by GabrielCamargoL**
- [x] general ticket update (accept all combination data <all/one/two/any/none fields in payload>)
- [x] go horse VAI CAVALA VAI CAVALA

### CODE REVIEW

File: src/main/java/com/helpduck/helpducktickets/entity/Ticket.java [hunk 1]
```diff
@@ -21,7 +25,12 @@ public class Ticket {
 	private String description;
 	private List<String> tags;
 	private Equipment equipment;
+
+	@Enumerated(EnumType.ORDINAL) // The default
 	private PriorityLevelEnum priorityLevel;
+
+	@Enumerated(EnumType.ORDINAL) // The default
+	@Formula("case status when 'awaiting' then 0 when 'underAnalysis' then 1 .when done then 2 end")
```
Comment (ThHenrique) [inline src/main/java/com/helpduck/helpducktickets/entity/Ticket.java:33]: formula do hamburgue de siri 🍔

File: src/main/java/com/helpduck/helpducktickets/model/hateoas/TicketHateoas.java [hunk 2]
```diff
@@ -53,6 +52,6 @@ public TicketHateoas(Ticket ticket) {
 		solution = ticket.getSolution();
 		createdAt = ticket.getCreatedAt();
 		updatedAt = ticket.getUpdatedAt();
-		concludedAt = ticket.getCreatedAt();
+		concludedAt = ticket.getConcludedAt();
```
Comment (ThHenrique) [inline src/main/java/com/helpduck/helpducktickets/model/hateoas/TicketHateoas.java:55]: hehe

**Review by ThHenrique — APPROVED** [review]
Mago, bruxo, maestro. Só perde pro R9

File: src/main/java/com/helpduck/helpducktickets/model/hateoas/TicketHateoas.java (hunk 2 above)
Comment (GabrielCamargoL) [reply src/main/java/com/helpduck/helpducktickets/model/hateoas/TicketHateoas.java:55]: :eyes:

**Pull request merged**